src/main/java/com/evolveum/polygon/connector/ldap/LdapUtil.java

```diff
@@ -245,29 +245,32 @@ public static boolean isOperationalAttribute(AbstractSchemaTranslator schemaTran
 		return false;
 	}
 
+	/**
+	 * Fetch a single entry using its DN.
+	 * 
+	 * @param connection The LDAP connection to use
+	 * @param dn The entry's DN
+	 * @param ldapObjectClass The entry's ObjectClass
+	 * @param options The options to use 
+	 * @param schemaTranslator The Schema translator instance
+	 * @return The found entry, or null if none is found.
+	 */
 	public static Entry fetchEntry(LdapNetworkConnection connection, String dn, 
 			org.apache.directory.api.ldap.model.schema.ObjectClass ldapObjectClass, 
 			OperationOptions options, AbstractSchemaTranslator schemaTranslator) {
 		String[] attributesToGet = getAttributesToGet(ldapObjectClass, options, schemaTranslator);
 		Entry entry = null;
-		LOG.ok("Search REQ base={0}, filter={1}, scope={2}, attributes={3}", 
```

Comment:
This actually has a good reason. All the log entries that describe LDAP operations have this format. So they are easy to find and understand in the log file.